cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -37,6 +37,7 @@ const (
 
 type serviceGenerateHelperInterface interface {
 	formatFullMethodName(service *protogen.Service, method *protogen.Method) string
```

Comment:
This is now called only once..  It should be fine to delete it and inline it into `genFullMethods` instead.

Reply:
Remved interface method and put directly in `genFullMethods` as suggested :)